specs/windows/interface_details.table

```diff
@@ -0,0 +1,33 @@
+table_name("interface_details")
```

Comment:
In practice things don't like up 100% across OSs, especially windows, as we are finding. Hiding columns would be a decent option so that SELECT * on windows does not look ugly on a table schema that was made for a POSIX system.